packages/web/components/Form/HelpModal.js

```diff
@@ -6,7 +6,7 @@ import PropTypes from 'prop-types';
 const Container = styled.div`
 	${({ theme: { colors } }) => css`
 		position: fixed;
-		z-index: 1000;
+		z-index: 1300;
```

Comment:
This increase was necessary so help modal overlaps header

![image](https://user-images.githubusercontent.com/13091635/98591570-304e6f80-22af-11eb-815b-de57f8b8b0ea.png)

Reply:
The header should be 1 and the modal should be 2, don't you think?

Reply:
@mateus4k just implemeting [material design standard](https://material.io/design/) about components elevation 👍

Reply:
Indeed, appbar should be 1100 instead of 1000.